cypress/e2e/ui-tests/test-project-as-contributor.cy.js

```diff
@@ -60,7 +60,27 @@ describe("Test project contributor user functionality", () => {
                 cy.get("title").should("have.text", project_title_name)
                 cy.get('h3').should('contain', project_name)
 
-                // TODO: add tests for all sections similar to Models below
+                // Check that the app limits work using Jupyter Lab as example
+                // step 1. create persistent volume
+                cy.get('[data-cy="create-app-card"]').contains('Persistent Volume').parent().siblings().find('.btn').click()
+                cy.get('input[name=app_name]').type("e2e-create-pv")
+                cy.get('.btn-primary').contains('Create').click()
+                // step 2. create 3 jupyter lab instances (current limit)
+                Cypress._.times(3, () => {
+                        cy.get('[data-cy="create-app-card"]').contains('Jupyter Lab').parent().siblings().find('.btn').click()
+                        cy.get('input[name=app_name]').type("e2e-create-jl")
+                        cy.get('.btn-primary').contains('Create').click()
+                  });
+                // step 3. check that the button to create another one does not work
+                cy.get('[data-cy="create-app-card"]').contains('Jupyter Lab').parent().siblings().find('.btn').should('not.have.attr', 'href')
+                // step 4. check that it is not possible to create another one using direct url
+                let projectURL
+                    cy.url().then(url => {
+                        projectURL = url
+                    });
+                cy.then(() =>
+                    cy.request({url: projectURL + "/apps/create/jupyter-lab?from=overview", failOnStatusCode: false}).its('status').should('equal', 403)
+                    )
```

Comment:
Great tests. This ties into my current branch aswell. I'm trying to create resources to be able to test the standard app. 
I think you should factor this code at make a new  `it("description", () => {`

I would suggest you make each of theses a new `it`.
1. Create PVC
2. Create Jupyter lab (and see that it gets deployed correctly
3. Create other stuff 
5. Test app limits by deploying more Jupyterlabs

Furthermore, in my branch, i've added a variable to `cypress.config.js` called `create_resources`
``` 
env: {
    do_reset_db: true,
    create_resources: true,
  },
```
and then i've added this idea to the tests themselves 

```
if (createResources === 'true') {
            # tests here

          } else {
            cy.log('Skipped because create_resources is not true');
          }
```
Just so we can turn of resource creation if the connection between docker compose and kubernetes is not working as expected on some machine.